server/activityItem/queries.ts

```diff
@@ -557,57 +558,32 @@ export const createPubDiscussionCommentAddedActivityItem = async (
 	});
 };
 
-export const createSubmissionActivityItem = async (
-	kind: 'submission-deleted' | 'submission-created',
-	actorId: null | string,
-	submissionId: string,
-) => {
-	const {
-		pubId,
-		pub: { title, communityId },
-	}: types.DefinitelyHas<types.Submission, 'pub'> = await Submission.findOne({
-		where: { id: submissionId },
-		include: [
-			{
-				model: Pub,
-				as: 'pub',
-			},
-		],
-	});
-	return createActivityItem({
-		actorId,
-		communityId,
-		pubId,
-		kind,
-		payload: {
-			submissionId,
-			pub: {
-				title,
-			},
-		},
-	});
-};
-
-export const createSubmissionStatusChangedActivityItem = async (
+export const createSubmissionStatusUpdatedActivityItem = async (
 	actorId: null | string,
 	submissionId: string,
 	oldSubmission: types.Submission,
 ) => {
-	const submission: types.DefinitelyHas<types.Submission, 'pub'> = await Submission.findOne({
-		where: { id: submissionId },
-		include: [
-			{
-				model: Pub,
-				as: 'pub',
-			},
-		],
-	});
+	const submission: types.DefinitelyHas<types.Submission, 'pub' | 'submissionWorkflow'> =
```

Comment:
It's news to me that we can put a `'prop1' | 'prop2'` into `DefinitelyHas`, and I frankly don't know what would result.